recipes/holopy/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ $(uname) == Darwin ]; then
+    export CFLAGS="${CFLAGS} -arch x86_64"
+    export FFLAGS="${FFLAGS} -static -ff2c -arch x86_64"
+    export LDFLAGS="${LDFLAGS} -Wall -undefined dynamic_lookup -bundle -arch x86_64"
+    export LDFLAGS="${LDFLAGS} -Wl,-search_paths_first -L$(pwd) $LDFLAGS"
+fi
```

Comment:
Can you try removing all of the stuff above? I don't think any of it is needed with `toolchain3`.